pkg/webhook/ingress/validating.go

```diff
@@ -155,6 +164,35 @@ func (r *handler) validateIngress(ctx context.Context, c client.Client, object I
 		return admission.Errored(http.StatusBadRequest, NewIngressClassForbidden(*ingressClass, *tnt.Spec.IngressClasses))
 	}
 
+	//TODO extract logic below into a method
```

Comment:
Maybe I should extract the logic into a method, since it's getting bigger. What do you think?

Reply:
Please, refer to the right name: Go doesn't provide __method__ but rather __struct functions__ :)

But I'd say webhook is quite atomic enough, no need to split into micro functions unless they're shared, but I'd say this is not the case.